tools/WinObjC.Tools/vsimporter-templates/packageable.project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "WinObjC.Packaging": "*",
+    "WinObjC.Frameworks": "*",
+    "WinObjC.Frameworks.UWP": "*",
+    "WinObjC.Frameworks.Core": "*",
+    "WinObjC.Language": "*",
+    "WinObjC.Logging": "*"
```

Comment:
Should it be all the nuget packages?  Or just the WinOjbc.Frameworks.UWP?  I thought there was a dependency chain.